File: src/public/api/v2/creators.php
<?php

use creator\Creator;

require_once $_SERVER['DOCUMENT_ROOT'] . '/../include/init.php';
header("content-type: application/json");

$creators = [];

foreach (Creator::cases() as $c) {
	$creators[] = [
		"id" => $c->value,
		"slug" => $c->slug(),
		"name" => $c->title(),
		"license" => $c->commonLicense()->slug(),
		"description" => $c->description(),
	];
}

echo json_encode($creators);

File: app/routes/api/creators.py
from fastapi import APIRouter
from fastapi.responses import JSONResponse

from app.creator.creator import Creator

router = APIRouter()


@router.get("/creators")
async def get_creators() -> JSONResponse:
    output = [
        {
            "id": c.value,
            "slug": c.slug(),
            "name": c.title(),
            "licenseUrl": c.license_url(),
            "description": c.description(),
        }
        for c in Creator
    ]
    return JSONResponse(content=output)
